vicinity/integrations/huggingface.py

```diff
@@ -131,8 +151,7 @@ def load_from_hub(cls, repo_id: str, token: str | None = None, **kwargs: Any) ->
             config = json.load(f)
             model_name_or_path = config.pop("model_name_or_path")
 
-        print(_MODEL_NAME_OR_PATH_PRINT_STATEMENT.format(model_name_or_path=model_name_or_path))
+        print(_MODEL_NAME_OR_PATH_PRINT_STATEMENT.format(model_name_or_path=model_name_or_path))  # noqa: T201
         backend_type = Backend(config["backend_type"])
         backend = get_backend_class(backend_type).load(repo_path / "backend")
-
-        return cls(items=items, backend=backend, metadata=config["metadata"], vector_store=vector_store)
+        return cls(items, vector_store, backend, config["metadata"])
```

Comment:
I think this typing is incorrect. Mixins are difficult to type, but this actually doesn't return a HuggingFaceHubMixin, it returns a vicinity instance.